src/java/generator/MapGenerator.java

```diff
@@ -60,6 +67,7 @@ public void save(String folderName, String mapName, SCMap map, long seed) {
 	}
 
 	public SCMap generate(long seed) throws ExecutionException, InterruptedException {
+		Biomes.load();
```

Comment:
if you want you could instead turn the load method into a static initializer, then it will run once the class is required the first time.